README.md

```diff
@@ -51,24 +51,116 @@ If BSG decide to create the ability to do this on live OR I receive a DCMA reque
 
 
 ## SPT-AKI Requirement
-* Stay in Tarkov works requires the [latest AKI Server](https://dev.sp-tarkov.com/SPT-AKI/Server) to run. You can learn about SPT-Aki [here](https://www.sp-tarkov.com/).
-* DO NOT INSTALL THIS ON TO SPT-Aki CLIENT! ONLY INSTALL THE SERVER!
```

Comment:
I refactored this section so that it remains a clear statement of the pre-requisite & warning, but points to the instructions below for the sake of clarity.